airbase-policy/src/main/resources/checkstyle/airbase-checks.xml

```diff
@@ -70,6 +70,18 @@
         <property name="format" value="^([^i]|i[^m]|im[^p]|imp[^o]|impo[^r]|impor[^t]|import[^ ]).*Math\.toIntExact" />
         <property name="message" value="Math.toIntExact should only be used with static imports" />
     </module>
+    <module name="RegexpSingleline">
+        <property name="format" value="^([^i]|i[^m]|im[^p]|imp[^o]|impo[^r]|impor[^t]|import[^ ]).*ImmutableMap\.toImmutableMap" />
```

Comment:
I know this is copied from above, but wonder if 
```
^(?!import).* ImmutableMap\.toImmutableMap
```
would be enough.

Reply:
I don't know. I thought that this way is for a reason.

Reply:
I believe that will work, but we can clean that up separately

Reply:
Filed https://github.com/airlift/airbase/issues/241 for that cleanup